cmd/kubeadm/app/cmd/init.go

```diff
@@ -56,7 +56,8 @@ var (
 		  mkdir -p $HOME/.kube
 		  sudo cp -i {{.KubeConfigPath}} $HOME/.kube/config
 		  sudo chown $(id -u):$(id -g) $HOME/.kube/config
-
+                  export KUBECONFIG=$HOME/.kube/admin.conf
```

Comment:
You don't need todo this.

Reply:
Yes. By default, `kubectl` uses `$HOME/.kube/config` as it's configuration file.